tidy3d/components/data/data_array.py

```diff
@@ -71,16 +71,23 @@ class DataArray(xr.DataArray):
     _data_attrs: Dict[str, str] = {}
 
     def __init__(self, data, *args, **kwargs):
-        """Initialize ``DataArray``."""
-
-        # initialize with untraced data
-        super().__init__(getval(data), *args, **kwargs)
-        # and put tracers in .attrs
+        super().__init__(data, *args, **kwargs)
         if isbox(data):
             self.attrs[AUTOGRAD_KEY] = data
```

Comment:
note: we could consider just adding this to a `self.autograd_tracers` attribute instead of attrs. not sure why I used attrs originally.